e2e/app/CreateAccountPage.ts

```diff
@@ -0,0 +1,218 @@
+  UNDERGRADUATE_STUDENT: 'Undergraduate (Bachelor level) student',
+  INDUSTRY: 'Industry',
+};
+
+export const EDUCATION_LEVEL = {
+  DOCTORATE: 'Doctorate',
+};
+
+export const FIELD_LABEL = {
+  READ_UNDERSTAND_PRIVACY_STATEMENT: 'I have read and understand the All of Us Research Program Privacy Statement',
+  READ_UNDERSTAND_TERMS_OF_USE: 'I have read and understand the All of Us Research Program Terms of Use described above',
+  INSTITUTION_NAME: 'Institution Name',
+  ARE_YOU_AFFILIATED: 'Are you affiliated with an Academic Research Institution',
+  RESEARCH_BACKGROUND: 'Please describe your research background, experience and research interests',
+  EDUCATION_LEVEL: 'Highest Level of Education Completed',
+  YEAR_OF_BIRTH: 'Year of Birth',
+};
+
+export default class CreateAccountPage extends BasePage {
+
+  public async getInvitationKeyInput(): Promise<TextBox> {
+    const textbox = new TextBox(this.puppeteerPage);
+    await textbox.withCss('#invitationKey');
+    return textbox;
+  }
+
+  public async getSubmitButton(): Promise<Button> {
+    const button = new Button(this.puppeteerPage);
+    await button.withLabel({text: 'Submit'});
+    return button;
+  }
+
+  public async getNextButton(): Promise<Button> {
+    const button = new Button(this.puppeteerPage);
+    await button.withLabel({text: 'Next'});
+    return button;
+  }
+
+  public async scrollToNthPdfPage(nth: number): Promise<ElementHandle> {
+    const selector = `.react-pdf__Document :nth-child(${nth}).react-pdf__Page.tos-pdf-page`;
+    const pdfPage = await this.puppeteerPage.waitForSelector(selector);
+    await this.puppeteerPage.evaluate(el => el.scrollIntoView(), pdfPage);
+    await this.puppeteerPage.waitFor(1000);
+    return pdfPage;
+  }
+
+  public async scrollToLastPdfPage(): Promise<ElementHandle> {
+    const selector = '.react-pdf__Document :last-child.react-pdf__Page.tos-pdf-page';
+    const pdfPage = await this.puppeteerPage.waitForSelector(selector);
+    await this.puppeteerPage.evaluate(el => el.scrollIntoView(), pdfPage);
+    await this.puppeteerPage.waitFor(1000);
+    return pdfPage;
+  }
+
+  public async getPrivacyStatementCheckbox(): Promise<CheckBox> {
+    const checkbox = new CheckBox(this.puppeteerPage);
+    await checkbox.withLabel({normalizeSpace: FIELD_LABEL.READ_UNDERSTAND_PRIVACY_STATEMENT});
+    return checkbox;
+  }
+
+  public async getTermsOfUseCheckbox(): Promise<CheckBox> {
+    const checkbox = new CheckBox(this.puppeteerPage);
+    await checkbox.withLabel({normalizeSpace: FIELD_LABEL.READ_UNDERSTAND_TERMS_OF_USE});
+    return checkbox;
+  }
+
+
+  public async getPrivacyStatementLabel(): Promise<Label> {
+    const label = new Label(this.puppeteerPage);
+    await label.withLabel({normalizeSpace: FIELD_LABEL.READ_UNDERSTAND_PRIVACY_STATEMENT});
+    return label;
+    // return this.getCheckboxLabel(FIELD_LABEL.READ_UNDERSTAND_PRIVACY_STATEMENT);
```

Comment:
Please remove commented out code

Reply:
Done. Also, looked elsewhere for commented out and removed them as well.